### ISSUE TITLE
Classification

### ISSUE THREAD

**Issue by sidhu1012** [Dacience/DataScientist#21]
Implement all classification algos present here
https://scikit-learn.org/stable/modules/classes.html#module-sklearn.linear_model

**Comment by bhargavasomya** [comment]
Hello! I would like to contribute to this project. Can you please help me get started?
Thanks

**Comment by sidhu1012** [comment]
>https://scikit-learn.org/stable/modules/classes.html#module-sklearn.linear_model



@bhargavasomya You have to make a new folder named classification in model and implement all of classification algos of sklearn library with test cases.

Same as done in regression.

**Comment by sidhu1012** [comment]
This is a big issue and should be resolved in multiple PRs

**Comment by soumya997** [comment]
sir, I have a question if you can clarify this, then it would very helpful.
Q. so this library is gonna be a rapper on top of sklearn, matplotlib and scipy etc. The syntax of these libraries are already simple and easy to use. And it's not like you are writing this from scratch and making the whole pipeline efficient. So, what is the motive behind writing a 3rd or a 4th party library?
I am really very sorry if I am wrong somewhere.

**Comment by Prachi0203** [comment]
@sidhu1012  Is there anything I can contribute to this issue. I am a beginner....:) 
Do I have to find mysellf what classification model I have to add to this?

**Comment by sidhu1012** [comment]
> Do I have to find mysellf what classification model I have to add to this?

Yes and since classification has just begun , you can start with easy ones like KNN.

**Comment by anishkabhatia** [comment]
@sidhu1012 are there any classification models left to be contributed. I am a beginner too!

**Comment by sidhu1012** [comment]
> @sidhu1012 are there any classification models left to be contributed. I am a beginner too!

Yes all are left just tell on which model you are gonna work and start working!

**Comment by anishkabhatia** [comment]
> > @sidhu1012 are there any classification models left to be contributed. I am a beginner too!
> 
> Yes all are left just tell on which model you are gonna work and start working!

I'm going forward with KNN

**Comment by Arneja12** [comment]
Working on Random Forest Classifier.

**Comment by Prachi0203** [comment]
I will work on Decision Trees

**Comment by sidhu1012** [comment]
> I will work on Decision Trees

When we have random forest then we don't need decision tree.

**Comment by Prachi0203** [comment]
okay Can I work on support vector machines?

**Comment by sidhu1012** [comment]
> okay Can I work on Logistic or linear regression? 

Linear regression is already done , we already  have a PR for logistic regression. Please take any other model.

**Comment by Arneja12** [comment]
Want to work upon Ridge classifier and ridge classifier CV.

**Comment by Arneja12** [comment]
Can i start working upon naive bayes models?
